test_scripts/API/SetAudioStreamingIndicator/ideas_SetAudioStreamingIndicator_conditions_for_SDL_to_handle_related_cases.txt

```diff
@@ -0,0 +1,79 @@
+./test_scripts/API/SetAudioStreamingIndicator/016_ATF_N_TC_SetAudioStreamingIndicator_param_wrongtype_non-media_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_with_wrongtype_non-media_app
+./test_scripts/API/SetAudioStreamingIndicator/017_ATF_N_TC_SetAudioStreamingIndicator_param_data_nonexisting_value_non-media_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_with_nonexisting_value_non-media_app
+./test_scripts/API/SetAudioStreamingIndicator/018_ATF_N_TC_SetAudioStreamingIndicator_param_empty_value_non-media_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_empty_value_non-media_app
+./test_scripts/API/SetAudioStreamingIndicator/019_ATF_N_TC_SetAudioStreamingIndicator_param_wrongtype_navi_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_with_wrongtype_navi_app
+./test_scripts/API/SetAudioStreamingIndicator/020_ATF_N_TC_SetAudioStreamingIndicator_param_data_nonexisting_value_navi_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_with_nonexisting_value_navi_app
+./test_scripts/API/SetAudioStreamingIndicator/021_ATF_N_TC_SetAudioStreamingIndicator_param_empty_value_navi_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_empty_value_navi_app
+./test_scripts/API/SetAudioStreamingIndicator/022_ATF_N_TC_SetAudioStreamingIndicator_param_wrongtype_comm_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_with_wrongtype_comm_app
+./test_scripts/API/SetAudioStreamingIndicator/023_ATF_N_TC_SetAudioStreamingIndicator_param_data_nonexisting_value_comm_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_with_nonexisting_value_comm_app
+./test_scripts/API/SetAudioStreamingIndicator/024_ATF_N_TC_SetAudioStreamingIndicator_param_empty_value_comm_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_empty_value_comm_app
+./test_scripts/API/SetAudioStreamingIndicator/025_ATF_N_TC_SetAudioStreamingIndicator_param_data_missing_parameter_media_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_with_nonexisting_value_media_app
+./test_scripts/API/SetAudioStreamingIndicator/026_ATF_N_TC_SetAudioStreamingIndicator_param_data_missing_parameter_non-media_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_with_nonexisting_value_non-media_app
```

Comment:
@istoimenova: Verification is the same as in 017. Seems has to be changed to: ...is_with_missing_parameter...